src/common/html-attributes/index.js

```diff
@@ -13,7 +13,13 @@ function camelToKebab(s) {
  */
 function processHtmlAttributes(input = {}) {
     const attributes = {};
-    const obj = input['*'];
+    const htmlAttributes = input.htmlAttributes;
+    const wildcardAttributes = input['*'];
+
+    let obj = htmlAttributes || wildcardAttributes;
+    if (htmlAttributes && wildcardAttributes) {
+        obj = Object.assign(wildcardAttributes, htmlAttributes);
```

Comment:
Is this getting babel-fied? (Since this might run client-side we might want to avoid `Object.assign` as it doesn't work in IE.)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Browser_compatibility

Reply:
We shouldn't be avoiding Object.assign, or other whitelisted areas of ES6. Let's add `babel-plugin-transform-object-assign` if we need it.